sql/analyzer/filters.go

```diff
@@ -52,6 +52,9 @@ func getFiltersByTable(n sql.Node) (filtersByTable, error) {
 			}
 			filters.merge(fs)
 		}
+		if o, ok := node.(sql.OpaqueNode); ok {
```

Comment:
This got mixed in here. Finding filter expressions across opaque nodes is not what we want.